docs/tutorials/01_basics/02_variables.md

```diff
@@ -0,0 +1,89 @@
+# Variables
+
+In order to store a value and re-use it later, it can be assigned to a variable.
+
+For example, if you want to say `Hello Penny!` three times, you don't need to repeat the same string multiple times.
+Instead, you can assign it to a local variable and re-use it:
+
+```{.crystal .crystal-play}
+message = "Hello Penny!"
+
+puts message
+puts message
+puts message
+```
+
+This program prints the string `Hello Penny!` three times to the standard output, each followed by a line break.
+
+The name of a local variable always starts with a lowercase [Unicode](https://en.wikipedia.org/wiki/Unicode) letter (or an underscore, but that's reserved for special use cases) and can otherwise consist of alphanumeric characters or underscores. As a typical convention, upper-case letter are avoided and names are written in `snake_case`.
```

Comment:
```suggestion
The name of a local variable always starts with a lowercase [Unicode](https://en.wikipedia.org/wiki/Unicode) letter (or an underscore, but that's reserved for special use cases) and can otherwise consist of alphanumeric characters or underscores. As a typical convention, upper-case letter are avoided and names are written in [`snake_case`](https://en.wikipedia.org/wiki/Snake_case).
```